access-control-app/int-test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -23,6 +24,36 @@
               (fixtures/grant-all-group-fixture ["PROV1" "PROV2"])
               (fixtures/grant-all-acl-fixture))
 
+(deftest permission-get-and-post-request-test
+  (let [save-basic-collection (fn [short-name]
+                                  (u/save-collection {:entry-title (str short-name " entry title")
+                                                      :short-name short-name
+                                                      :native-id short-name
+                                                      :provider-id "PROV1"}))
+        coll1 (save-basic-collection "coll1")
+        coll2 (save-basic-collection "coll2")
+        coll3 (save-basic-collection "coll3")
+        coll4 (save-basic-collection "coll4")
+        post-data-body (str "user_type=guest"
+                         "&concept_id=" coll1
+                         "&concept_id=" coll2
+                         "&concept_id=" coll3
+                         "&concept_id=" coll4)]
+    (testing "permissions endpoint allows post request"
+      (let [response (client/post (ac/acl-permission-url
+                                    (transmit-config/context->app-connection
+                                     (u/conn-context)
+                                     :access-control))
+                       {:basic-auth ["user" "pass"]
+                        :body post-data-body
+                        :content-type "application/x-www-form-urlencoded"})
+            body (get response :body)]
+
+        (is (string/includes? body coll1))
```

Comment:
These checks are too opaque. We should either add a `get-permission-via-post` call in all the existing tests and verify they all return the same result as `get` call or in this test makes a get permissions call via `get` and verify the responses from `post` and `get` match at the least.

Reply:
or remove this test and add a test case in at least one of the tests below to verify that `get` and `post` with the same query params return the same result.

Reply:
I played with each option and decided to go with adding the `get` response to this test.